test/fixtures/max-len.pass1.js

```diff
@@ -0,0 +1 @@
+var commenting = 'me after'; // is A ok even if really loooooooooooooooooooooooooooooooooooooooooooooooooooong
```

Comment:
hmm... super-long comments still require scrolling... 
but the chances people will leave long comments is _low_ so. 👍